core/src/apps/common/coininfo.py.mako

```diff
@@ -144,6 +144,9 @@ coins_alt_t2t1 = [c for c in supported_on("T2T1", bitcoin) if c.name not in btc_
 coins_btc_t2b1 = [c for c in supported_on("T2B1", bitcoin) if c.name in btc_names]
 coins_alt_t2b1 = [c for c in supported_on("T2B1", bitcoin) if c.name not in btc_names]
 
+coins_btc_t3t1 = [c for c in supported_on("T3T1", bitcoin) if c.name in btc_names]
+coins_alt_t3t1 = [c for c in supported_on("T3T1", bitcoin) if c.name not in btc_names]
```

Comment:
....whiiiich you can then use to implement the TODO here, and change the code generator iterate over model names